tests/components/tesla_fleet/fixtures/history_status.json

```diff
@@ -0,0 +1,45 @@
+{
+  "response": {
+    "period": "day",
+    "time_series": [
+      {
+        "timestamp": "2023-06-01T01:00:00-07:00",
+        "solar_energy_exported": 70940,
+        "generator_energy_exported": 0,
+        "grid_energy_imported": 521,
+        "grid_services_energy_imported": 17.53125,
+        "grid_services_energy_exported": 3.80859375,
+        "grid_energy_exported_from_solar": 43660,
+        "grid_energy_exported_from_generator": 0,
+        "grid_energy_exported_from_battery": 19,
+        "battery_energy_exported": 10030,
+        "battery_energy_imported_from_grid": 80,
+        "battery_energy_imported_from_solar": 16800,
+        "battery_energy_imported_from_generator": 0,
+        "consumer_energy_imported_from_grid": 441,
+        "consumer_energy_imported_from_solar": 10480,
+        "consumer_energy_imported_from_battery": 10011,
+        "consumer_energy_imported_from_generator": 0
```

Comment:
This fixture appears to be missing the `total_*` values which are only returned when their values are >0, is that intentional?

Reply:
That's interesting, I think that Tesla has updated their Fleet API from when they published their documentation because that test file is just copied from the documentation with values changed, but it shouldn't matter because those values are not used in the integration.

Reply:
I guess my point is if we use those values it greatly simplifys things because you dont need to add all the other values together and can use the totals directly.